diff --git a/po/Makefile.in.in.orig b/po/Makefile.in.in
index b98c676..406b240 100644
--- a/po/Makefile.in.in.orig
+++ b/po/Makefile.in.in
@@ -32,7 +32,7 @@ MKINSTALLDIRS = @MKINSTALLDIRS@
 
 # KG: tried to find ../.././/mkinstalldirs ???
 #mkinstalldirs = $(SHELL) `case "$(MKINSTALLDIRS)" in /*) echo "$(MKINSTALLDIRS)" ;; *) echo "$(top_builddir)/$(MKINSTALLDIRS)" ;; esac`
-mkinstalldirs = .././mkinstalldirs
+mkinstalldirs = @top_srcdir@/mkinstalldirs
 
 CC = @CC@
 GMSGFMT = @GMSGFMT@
